### ISSUE TITLE
DOC: Missing put_writes and aput_writes Method in "Create a Custom Checkpointer Using Redis"

### ISSUE THREAD

**Issue by GuilhermeKO** [langchain-ai/langgraph#1134]
### Issue with current documentation:

In the documentation "Create a Custom Checkpointer Using Redis"  there is no implementation of the put_writes and aput_writes methods.

This is causing the following error: NotImplementedError: This method was added in langgraph 0.1.7. Please update your checkpoint saver to implement it.

link to referred doc: https://langchain-ai.github.io/langgraph/how-tos/persistence_redis/

### Idea or request for content:

Request implementation of put_writes and a_putwrites methods on persistence_redis.ipynb.
https://github.com/langchain-ai/langgraph/blob/main/examples/persistence_redis.ipynb?short_path=e42963f

Both methods are implemented in persistence_postgres.ipynb for example:
https://github.com/langchain-ai/langgraph/blob/main/examples/persistence_postgres.ipynb?short_path=5874aa8
```
def put_writes(
        self,
        config: RunnableConfig,
        writes: Sequence[Tuple[str, Any]],
        task_id: str,
    ) -> None:
        with self._get_sync_connection() as conn:
            with conn.cursor() as cur:
                cur.executemany(
                    self.UPSERT_WRITES_QUERY,
                    [
                        (
                            str(config["configurable"]["thread_id"]),
                            str(config["configurable"]["thread_ts"]),
                            task_id,
                            idx,
                            channel,
                            self.serde.dumps(value),
                        )
                        for idx, (channel, value) in enumerate(writes)
                    ],
                )
            conn.commit()

async def aput_writes(
        self,
        config: RunnableConfig,
        writes: Sequence[Tuple[str, Any]],
        task_id: str,
    ) -> None:
        async with self._get_async_connection() as conn:
            async with conn.cursor() as cur:
                await cur.executemany(
                    self.UPSERT_WRITES_QUERY,
                    [
                        (
                            str(config["configurable"]["thread_id"]),
                            str(config["configurable"]["thread_ts"]),
                            task_id,
                            idx,
                            channel,
                            self.serde.dumps(value),
                        )
                        for idx, (channel, value) in enumerate(writes)
                    ],
                )
            await conn.commit()
```

**Comment by pranitsawant80** [comment]
the above method is not working for redis.   The above code only works for postgres. I am still facing same error.. please let me know if you have any solution for it

**Comment by changyuan** [comment]
the above method is not working for redis.

**Comment by Glinte** [comment]
No one said they work for redis

**Comment by DhavalThkkar** [comment]
@Glinte check the latest issue #1221. I have updated the code if you'd like to use

**Comment by vbarda** [comment]
Fixed in #1286
